mail/api.py

```diff
@@ -157,14 +176,45 @@ def send_financial_aid_email(cls, acting_user, financial_aid, subject, body):
         Returns:
             requests.Response: response from Mailgun
         """
-        response = cls.send_individual_email(subject, body, financial_aid.user.email)
+        from_address = cls.default_params()['from']
+        to_address = financial_aid.user.email
+        response = cls.send_individual_email(subject, body, to_address)
         if response.status_code == HTTP_200_OK:
             FinancialAidEmailAudit.objects.create(
                 acting_user=acting_user,
                 financial_aid=financial_aid,
-                to_email=financial_aid.user.email,
-                from_email=cls.get_base_params()['from'],
+                to_email=to_address,
+                from_email=from_address,
                 email_subject=subject,
                 email_body=body
             )
         return response
+
+    @classmethod
+    def send_course_team_email(cls, user, course, subject, body):
+        """
+       Sends a text email from a user to a course team.
+
+       Args:
+            user (User): A User
+            course (courses.models.Course): A Course
+            subject (str): Email subject
+            body (str): Email body
+        Returns:
+            requests.Response: HTTP Response from Mailgun
+       """
+        if not course.contact_email:
+            message = (
+                'Course team contact email attempted for course without contact_email'
+                '(id: {}, title: "{}")'.format(course.id, course.title)
+            )
+            log.error(message)
+            raise ImproperlyConfigured(message)
```

Comment:
there are other parts of the codebase that follow this pattern (eg: `financialaid/api.py` lines 40, 111). are you sure that we don't need to log the error explicitly?